### ISSUE TITLE
control scheme

### ISSUE THREAD

**Issue by z2oh** [z2oh/MegaManClone#3]
we should probably stick to unity's input system (https://docs.unity3d.com/Manual/ConventionalGameInput.html) for input to our game; this will allow us to use pretty much any plug-n-play controller with our game right out of the box (I hope). right now we are using horizontal axis for movement and 'space' for jump. space needs to be moved to an input manager button, and we need to figure out the best default mappings for 'fire' and 'switch gravity/activate'. Any ideas?

**Comment by lewaxml** [comment]
In the script I'm working on I've been using GetInput or GetAxis for all input which is what I believe you are saying. 

Are we going to have a 'switch gravity' button? I though that was going to be based on environmental factors such as shooting a switch.

**Comment by z2oh** [comment]
@lewaxml Yes the environmental factors will be largely responsible for changing the gravity. I figured we might have a lever or something that we need to 'activate' to switch the gravity or something. That's more dependent on level design though.
